src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    protected String machine;

```

Comment:
The field 'machine' is somewhat ambiguous. Consider renaming it to something more descriptive that reflects its purpose in the context of a 'Machine', such as 'machineType' or 'machineName'.